WebdriverJs-Mocha/tests/performance.js

```diff
@@ -0,0 +1,98 @@
+const assert = require('assert');
+const { Builder } = require('selenium-webdriver');
+const SauceLabs = require('saucelabs');
+
+const username = process.env.SAUCE_USERNAME;
+const accessKey = process.env.SAUCE_ACCESS_KEY;
+const saucelabs = new SauceLabs({
+	username,
+	password: accessKey,
+});
+
+const server = `http://${username}:${accessKey}@ondemand.saucelabs.com:80/wd/hub`;
+const capabilities = {
+	platform: 'OS X 10.10',
+	version: '70.0',
+	browserName: 'chrome',
+	extendedDebugging: true,
+	crmuxdriverVersion: 'stable',
```

Comment:
I don't think we need to include the crumxdriverVersion with the sample script 'stable' should be the default.